optional_plugins/result_upload/avocado_result_upload/__init__.py

```diff
@@ -84,7 +84,7 @@ def configure(self, parser):
             def_ssh = ('ssh -oLogLevel=error -o stricthostkeychecking=no'
                        ' -o userknownhostsfile=/dev/null'
                        ' -o batchmode=yes -o passwordauthentication=no')
-            def_upload_cmd = 'rsync -arz -e \'%s \'' % def_ssh
+            def_upload_cmd = '%s -arz -e \'%s \'' % (rsync_bin, def_ssh)
```

Comment:
definitely, it could actually lead to an error as `find_command` searches additional PATHs.